src/Type.h

```diff
@@ -683,12 +690,8 @@ class RecordType final : public Type
 
 	void AddFieldsDirectly(const type_decl_list& types, bool add_log_attr = false);
 
-	/**
-	 *
-	 * Populates a new instance of the record with its initial values.
-	 * @param r  The record's underlying value vector.
-	 */
-	void Create(std::vector<std::optional<ZVal>>& r) const;
+	const auto& FieldInits() const { return field_inits; }
```

Comment:
Could you perhaps add some documentation for the two getter functions here? I know there is a comment below that explains this a bit - but it is easy to miss this, when just looking at the functions.

Reply:
Likewise, sounds good.

Are there more reviewing comments potentially coming (a question also for @timwoj), or should I go ahead with the commenting updates?